pkg/networkservice/chains/nsmgr/select_forwarder_test.go

```diff
@@ -219,8 +219,6 @@ func Test_DiscoverForwarder_ChangeForwarderOnDeath_LostHeal(t *testing.T) {
 	conn, err = nsc.Request(ctx, request.Clone())
 	require.NoError(t, err)
 	require.Equal(t, 1, counter.UniqueRequests())
-	require.Equal(t, 2, counter.Requests())
-	require.Equal(t, 0, counter.Closes())
```

Comment:
Could you explain the change?

Reply:
I modified it because this test was unstable, and yesterday the CI was constantly failing.
There was somewhat of a race between the test code and healing.
I figured out that this check isn't really important for the purpose of this test.
Now I pushed a better solution which makes the test more deterministic.